sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java

```diff
@@ -248,7 +251,7 @@ public Read withKeyRange(ByteKeyRange keyRange) {
      */
     public Read withTableId(String tableId) {
       checkNotNull(tableId, "tableId");
-      return new Read(options, tableId, keyRange, filter, bigtableService);
+      return new Read(options, StaticValueProvider.of(tableId), keyRange, filter,null, null, bigtableService);
```

Comment:
We should update the Javadoc for the Read/Write transform to indicate what can be parameterized, etc.

Specifically, we should explain the rationale between what was made parameterizable and not.